modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_ar.properties

```diff
@@ -3707,6 +3707,7 @@ commerce-order-rule-commerce-qualifier-configuration-name=مؤهلات قاعد
 commerce-order-rule-entry-configuration-name=قواعد الأمر
```

Comment:
Move commits that contain auto-generated files at the end of the history

Reply:
I'll move it once I push the commits with the fixes you requested. Thanks

Reply:
Done